packages/marketplace/src/components/pages/admin-dev-management/admin-dev-management.tsx

```diff
@@ -162,34 +184,12 @@ export const onPageChangeHandler = (history: History<any>, queryParams: AdminDev
   return history.push(`${Routes.ADMIN_DEV_MANAGEMENT}/${page}${query}`)
 }
 
-export const onSearchHandler = (history: History<any>, match: { params: { page?: number } }) => (
+export const onSearchHandler = (history: History<any>, page: number) => (
   queryParams: AdminDevManagementFilterFormValues,
 ) => {
-  const page = match.params.page || 1
   const query = setQueryParams(queryParams) ? `?${setQueryParams(queryParams)}` : ''
```

Comment:
should modify the function setQueryParams if queryParams is `{}` or `null` or `undefined` return `''` instead of check when use it